e2e/suites/actions-available/files-folders/test-data.ts

```diff
@@ -55,10 +56,21 @@ const fileSharedFavLockedContextMenu = [
   'Copy',
   'Delete',
   'Manage Versions',
+  'Edit Aspects',
```

Comment:
I'm pretty sure this is not ok. User should not be able to Edit Aspects if the file is locked